autocomplete/definitions/namedTypes/tes3magicSourceInstance/playVisualEffect.lua

```diff
@@ -0,0 +1,11 @@
+return {
+	type = "method",
+	description = "This function plays an effect from tes3magicSourceInstance of a given index.",
+	arguments = {
```

Comment:
This function takes a table as arguments, so you'll need to use `tableParams` here. See other table-accepting functions for examples.

On the C side, recognizing this can be done by hitting F12 on the binding function's name. You'll see something like:
```cpp
void MagicSourceInstance::playSpellVFX_lua(sol::table params)
```

This means that it is only accepting a table. You would then have to look for where `params` is used in the code in the function to see what values are available and what their default values are.